python_ta/patches/transforms.py

```diff
@@ -32,7 +32,7 @@ def new_get_ast(self, filepath, modname, data):
             else:
                 ast.accept(CFGVisitor())
         except Exception as e:
-            logging.warning(f"Could not run Z3Visitor: {e}")
+            logging.warning(f"Could not run CFGVisitor: {e}")
```

Comment:
This change is good, and can be documented as a separate entry in the changelog (you can call it a "bug fix" even though it's only improving the error message).